### DISCUSSION TITLE
How hard would it be to adapt this to sqlite?

### DISCUSSION THREAD

**Discussion by ge** [sqlpage/SQLPage#30] [General]
I know, I know.  I'm a weenie.  But I'm working with an existing sqlite database produced by an outside source.  I guess I'd have to learn Rust too, but how different is the database connector?

Thanks,
ge, chapel hill, nc

**Comment by ge** [comment]
Oh!  I see that it will work with sqlite!  I got the wrong idea from seeing the announcement on the postgreSQL website.

Now.  Can I figure this thing out?  We'll see.

**Comment by lovasoa** [comment]
Hi ! 

Yes, it currently supports SQLite, PostgreSQL, and MySQL. SQLite is even the default.

>  Can I figure this thing out? We'll see.

Please post here if you encounter any difficulty ! And I'd love to get feedback about the tutorial and documentation too. Don't hesitate to complain :stuck_out_tongue:  !
